Control/public/about/About.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+*/
+
+import {Observable, RemoteData} from '/js/src/index.js';
+
+const INTEGRATED_SERVICE_LABEL = 'INTEG_SERVICE';
+
+/**
+ * Model representing About page
+ */
+export default class About extends Observable {
+  /**
+   * Initialize services with their keys and corresponding paths
+   * @param {Model} model
+   */
+  constructor(model) {
+    super();
+
+    this.model = model;
+
+    this.services = {
+      apricot: 'apricot',
+      core: 'core',
+      consul: 'consul',
+      grafana: 'grafana',
+      gui: 'gui',
+      [INTEGRATED_SERVICE_LABEL]: 'core/services',
+      notification: 'notification',
+    };
+
+    this.statuses = {};
+  }
+
+  /**
+   * Load AliECSGUI and its dependencies data in control-remoteData
+   * @returns {void}
+   */
+  async retrieveInfo() {
+    for (const key in this.services) {
+      this.retrieveServiceState(key, this.services[key])
+    }
+    this.retrieveWsInfo();
+  }
+
+  /**
+   * Return a list of services that are in error state being either:
+   * * RemoteData.failure - case in which the name and message is returned only
+   * * Service in configured state but in failure - case in which full payload is returned
+   * @returns {Array<{name, message}|Service>} - list of services
+   */
+  getInErrorServices() {
```

Comment:
Wouldn't it be best to fill 4 list when retrieving data with a serie of `if` instead of filtering them when getting?